### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
## Issue(s)
#489 

## Description
Add a note on JDK prerequisite to README.

## Category

- [ ] Bugfix
- [ ] Enhancement
- [ ] New Feature
- [ ] Configuration Change
- [ ] Versioning Change
- [ ] Unit Test(s)
- [x] Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,12 @@ If you run into problems using the SDK, you can
 * Post [issues][github-issues] here on GitHub (for code errors)
  
 ## Getting started
- 
+
+### Prerequisites
+
+* JDK 8 or later
+* Apache Maven (3.6.x or later)
```
Comment (bdemers) [inline README.md:60]: Any build system could be used, gradle, ivy, etc. (though we likely would only support Maven or Gradle)

Comment (arvindkrishnakumar-okta) [reply README.md:60]: so we'll add a "recommended" next to Maven?

Comment (bdemers) [reply README.md]: I'd just leave the build system off completely, or maybe just say the artifacts are published to [Maven Central](https://search.maven.org/search?q=g:com.okta.sdk)

(it's similar to the Node JS world, you could use yarn or NPM, it's _mostly_ an implementation detail)

Comment (arvindkrishnakumar-okta) [reply README.md]: done

**Review by bdemers — APPROVED** [review]

**Pull request merged**